Sources/CrossCompilationDestinationsTool/RemoveDestination.swift

```diff
@@ -17,8 +17,8 @@ import PackageModel
 
 import struct TSCBasic.AbsolutePath
 
-struct RemoveDestination: DestinationCommand {
-    static let configuration = CommandConfiguration(
+public struct RemoveDestination: DestinationCommand {
```

Comment:
curious why these need to come public?

Reply:
Clarified in the PR to the `main` branch, this is due to `SwiftDestinationTool` referencing this type moving to a different target, while `RemoveDestination` could stay where it were.